.travis.yml

```diff
@@ -8,3 +8,9 @@ install: mvn -e install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 
 jdk:
   - oraclejdk8
+  
+deploy:
+  provider: bintray
+  file: descriptor.json
+  user: yglazko
+  key: a056771a462c6051c1074e7e6791dc951f1da36f
```

Comment:
Is the key encrypted?

Looking at [1](http://stackoverflow.com/q/32356238) it seems as if the key isn't encrypted and if it were encrypted it would not work.